packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -75,10 +89,10 @@ export class ColorPicker extends React.Component<IColorPickerProps, any> {
             <tbody>
               <tr>
                 <td><TextField className='ms-ColorPicker-input' value={ color.hex } /></td>
-                <td style={ { width: '18%' } }><TextField className='ms-ColorPicker-input' value={ color.r } /></td>
-                <td style={ { width: '18%' } }><TextField className='ms-ColorPicker-input' value={ color.g } /></td>
-                <td style={ { width: '18%' } }><TextField className='ms-ColorPicker-input' value={ color.b } /></td>
-                <td style={ { width: '18%' } }><TextField className='ms-ColorPicker-input' value={ color.a } /></td>
+                <td style={ { width: '18%' } }><TextField className='ms-ColorPicker-input' value={ String(color.r) } /></td>
```

Comment:
I think this shouldn't be an issue given current `getColorFromString` throws exception if color string is not proper which ends the react cycle and doesn't result in a re-render.